project/Dependencies.scala

```diff
@@ -91,119 +337,163 @@ object Dependencies {
-    "com.typesafe.akka" %% "akka-multi-node-testkit" % AkkaVersion % "test",
-    "com.typesafe.akka" %% "akka-stream-testkit" % AkkaVersion % "test",
+    akkaClusterTools,
+    akkaTestkit % Test,
+    akkaMultiNodeTestkit % Test,
+    akkaStreamTestkit % Test,
     scalaTest % Test
   )
 
   val `persistence-core` = libraryDependencies ++= Seq(
     scalaJava8Compat,
-    "com.typesafe.akka" %% "akka-persistence" % AkkaVersion,
-    "com.typesafe.akka" %% "akka-persistence-query-experimental" % AkkaVersion,
-    "com.typesafe.akka" %% "akka-cluster-sharding" % AkkaVersion,
-    "com.typesafe.play" %% "play" % PlayVersion,
-    "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test",
-    "com.typesafe.akka" %% "akka-multi-node-testkit" % AkkaVersion % "test",
-    "com.typesafe.akka" %% "akka-stream-testkit" % AkkaVersion % "test",
+    scalaXml,
+    guava,
+    akkaPersistence,
+    akkaPersistenceQuery,
+    akkaClusterSharding,
+    akkaSlf4j,
+    play,
+    akkaTestkit % Test,
+    akkaMultiNodeTestkit % Test,
+    akkaStreamTestkit % Test,
     scalaTest % Test,
-    "com.novocode" % "junit-interface" % "0.11" % "test"
+    "com.novocode" % "junit-interface" % "0.11" % Test,
+
+    // These dependencies get upgraded in Test so that we can ensure that our tests are using the same dependencies
+    // as the users will have in their tests
+    "com.lmax" % "disruptor" % "3.3.6" % Test,
+    "javax.validation" % "validation-api" % "1.1.0.Final" % Test,
+    "org.hibernate" % "hibernate-validator" % "5.2.4.Final" % Test,
+    jbossLogging % Test,
+    "org.slf4j" % "log4j-over-slf4j" % "1.7.21" % Test,
+    "org.xerial.snappy" % "snappy-java" % "1.1.2.6" % Test,
+    "org.yaml" % "snakeyaml" % "1.16" % Test,
+    "com.fasterxml" % "classmate" % "1.3.0" % Test
   )
 
   val `persistence-javadsl` = libraryDependencies ++= Nil
 
   val `persistence-scaladsl` = libraryDependencies ++= Nil
 
   val `persistence-cassandra-core` = libraryDependencies ++= Seq(
-    "com.typesafe.akka" %% "akka-persistence-cassandra" % AkkaPersistenceCassandraVersion,
-    "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion % "test" exclude("io.netty", "netty-all"),
-    "io.netty" % "netty-codec-http" % NettyVersion % "test",
-    "io.netty" % "netty-transport-native-epoll" % NettyVersion % "test" classifier "linux-x86_64"
+    akkaPersistenceCassandra,
+
+    // cassandra-driver-core pulls in an older version of all these
+    "io.netty" % "netty-buffer" % NettyVersion,
+    "io.netty" % "netty-codec" % NettyVersion,
+    "io.netty" % "netty-common" % NettyVersion,
+    "io.netty" % "netty-handler" % NettyVersion,
+    "io.netty" % "netty-transport" % NettyVersion,
+
+    "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion % Test exclude("io.netty", "netty-all"),
+    "io.netty" % "netty-codec-http" % NettyVersion % Test,
+    "io.netty" % "netty-transport-native-epoll" % NettyVersion % Test classifier "linux-x86_64"
```

Comment:
Total red herring, JNA is actually getting bumped by this:

```
├─ org.apache.cassandra:cassandra-all:3.0.9
...
|  ├─ org.caffinitas.ohc:ohc-core:0.4.3
|  |  ├─ com.google.guava:guava:16.0 -> 19.0
|  |  ├─ net.java.dev.jna:jna:4.1.0
|  |  └─ org.slf4j:slf4j-api:1.7.12 -> 1.7.21
```

The problem is that Coursier is resolving things slightly differently. When I disable Coursier, it passes. That's a bummer because Coursier is much faster.